AWSSDKSwiftCLI/Package.swift

```diff
@@ -5,28 +5,61 @@ import PackageDescription
 let package = Package(
     name: "AWSSDKSwiftCLI",
     platforms: [
-        .macOS(.v10_15)
+        .macOS(.v13)
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser", from: "1.2.0"),
         .package(url: "https://github.com/apple/swift-package-manager", from: "0.6.0"),
         .package(url: "https://github.com/apple/swift-algorithms", from: "1.0.0"),
         .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
+        .package(url: "https://github.com/awslabs/aws-sdk-swift", from: "0.46.0"),
     ],
     targets: [
         .executableTarget(
             name: "AWSSDKSwiftCLI",
             dependencies: [
+                "AWSCLIUtils",
                 .product(name: "ArgumentParser", package: "swift-argument-parser"),
-                .product(name: "PackageDescription", package: "swift-package-manager"),
                 .product(name: "Algorithms", package: "swift-algorithms"),
-                .product(name: "Logging", package: "swift-log"),
             ],
             resources: [
                 .process("Resources/Package.Base.swift"),
                 .process("Resources/DocIndex.Base.md")
             ]
         ),
+        .executableTarget(
+            name: "spr-publish",
+            dependencies: [
+                "SPR",
+                "AWSCLIUtils",
+                .product(name: "ArgumentParser", package: "swift-argument-parser"),
+            ]
+        ),
+        .executableTarget(
+            name: "spr-multi-publish",
+            dependencies: [
+                "SPR",
+                "AWSCLIUtils",
+                "spr-publish",
+            ]
+        ),
+        .target(
```

Comment:
The `SPR` target is used for sharing registry-related code between the CLI commands.